.github/workflows/behat-update-path-stability-1.yml

```diff
@@ -88,4 +88,4 @@ jobs:
       # Run the upgrade path behat tests
       - name: Run Integration tests
         run: |
-          docker exec -i social_ci_behat sh /var/www/scripts/social/behatstability.sh "stability-1 --stop-on-failure --strict  --colors"
+          docker exec -i social_ci_behat sh /var/www/scripts/social/behatstability.sh "stability-1&&~DS-1136&&~DS-3605 --stop-on-failure --strict  --colors"
```

Comment:
We don't need DS-1136 and DS-3605 as well. They are being covered in other tests, also they don't add a lot of value in here.